@jpkrohling looks like examples failed to build:

```
ERROR: Unable to lock database: Permission denied
ERROR: Failed to open apk database: Permission denied
ERROR: Service 'otel-collector' failed to build : The command '/bin/sh -c apk --update add ca-certificates' returned a non-zero code: 99
make: *** [Makefile:237: build-examples] Error 1
```

--- a/cmd/otelcontribcol/Dockerfile
+++ b/cmd/otelcontribcol/Dockerfile
@@ -2,8 +2,12 @@ FROM alpine:latest as certs
 RUN apk --update add ca-certificates
 
 FROM scratch
+
+ARG USER_UID=10001
+USER ${USER_UID}
+
 COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 COPY otelcontribcol /
-EXPOSE 55680 55679
+EXPOSE 4317 55680 55679
 ENTRYPOINT ["/otelcontribcol"]
 CMD ["--config", "/etc/otel/config.yaml"]

--- a/examples/tracing/Dockerfile
+++ b/examples/tracing/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.14 AS build
+FROM golang:1.16 AS build
 
 WORKDIR /src
 ADD . /src
@@ -9,7 +9,11 @@ FROM alpine:latest as certs
 RUN apk --update add ca-certificates
 
 FROM scratch
+
+ARG USER_UID=10001
+USER ${USER_UID}
+
 COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 COPY --from=build /src/bin/otelcontribcol_linux_amd64 /otelcontribcol
 ENTRYPOINT ["/otelcontribcol"]
-EXPOSE 55680 55679
+EXPOSE 4317 55680 55679

--- a/exporter/loadbalancingexporter/example/Dockerfile
+++ b/exporter/loadbalancingexporter/example/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.14 AS build
+FROM golang:1.16 AS build
 
 WORKDIR /src
 ADD . /src
@@ -9,7 +9,11 @@ FROM alpine:latest as certs
 RUN apk --update add ca-certificates
 
 FROM scratch
+
+ARG USER_UID=10001
+USER ${USER_UID}
+
 COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 COPY --from=build /src/bin/otelcontribcol_linux_amd64 /otelcontribcol
 ENTRYPOINT ["/otelcontribcol"]
-EXPOSE 55680 55679
+EXPOSE 4317 55680 55679

--- a/exporter/lokiexporter/example/Dockerfile
+++ b/exporter/lokiexporter/example/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.14 AS build
+FROM golang:1.16 AS build
 
 WORKDIR /src
 ADD . /src
@@ -9,7 +9,11 @@ FROM alpine:latest as certs
 RUN apk --update add ca-certificates
 
 FROM scratch
+
+ARG USER_UID=10001
+USER ${USER_UID}
+
 COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 COPY --from=build /src/bin/otelcontribcol_linux_amd64 /otelcontribcol
 ENTRYPOINT ["/otelcontribcol"]
-EXPOSE 55680 55679
+EXPOSE 4317 55680 55679

--- a/exporter/splunkhecexporter/example/Dockerfile
+++ b/exporter/splunkhecexporter/example/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.14 AS build
+FROM golang:1.16 AS build
 
 WORKDIR /src
 ADD . /src
@@ -9,7 +9,11 @@ FROM alpine:latest as certs
 RUN apk --update add ca-certificates
 
 FROM scratch
+
+ARG USER_UID=10001
+USER ${USER_UID}
+
 COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 COPY --from=build /src/bin/otelcontribcol_linux_amd64 /otelcontribcol
 ENTRYPOINT ["/otelcontribcol"]
-EXPOSE 55680 55679
+EXPOSE 4317 55680 55679

--- a/receiver/simpleprometheusreceiver/examples/federation/prom-counter/Dockerfile
+++ b/receiver/simpleprometheusreceiver/examples/federation/prom-counter/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.14-stretch
+FROM golang:1.16-stretch
 
 WORKDIR /go/src/app
 

